src/components/Book.js

```diff
@@ -1,12 +1,11 @@
 import PropTypes from 'prop-types';
```

Comment:
README should have a project title, project summary, and set of instructions to run the project locally.